.github/workflows/peer-api.yaml

```diff
@@ -19,4 +19,4 @@ jobs:
         run: npm ci
 
       - name: Check API dependency semantics
-        run: npx lerna exec "node \$LERNA_ROOT_PATH/scripts/peer-api-check.js"
+        run: node ./scripts/peer-api-check.js
```

Comment:
note for reviewer: this script was used only here so now it's changed to walk over all packages to do the API check.